docs/source/models/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -3819,23 +3819,22 @@ opportunity for duplication. Since this mechanism occurs within the
 simulation, there is a natural maximum that we will impose in the
 noise function.
 
-Guardian-based duplication is applied to three mutually exclusive categories of
+Guardian-based duplication is applied to two mutually exclusive categories of
 simulants based on age and GQ status: Simulants younger than 18  (<18) and not
-in GQ; simulants 18-23 (18 <= age < 24) and not in GQ; and simulants under 24
-(<24) and in GQ.
+in GQ and simulants under 24 (<24) and in college GQ.
 
-For each of the three categories of simulants, the maximum duplication rate will
+For each of the two categories of simulants, the maximum duplication rate will
 be calculated based on those who have a guardian living at a different address
 in the sim. Note that all simulants in *college* GQ are initialized with a
 guardian living at a different address, but this is not true for simulants in
-other types of GQ, so all three maximum duplication rates will be less than
+other types of GQ, so both maximum duplication rates will be less than
```

Comment:
Since we no longer duplicate simulants in GQ other than college, this could be reworded a bit.

I suggest changing "but this is not true for simulants in other types of GQ" to "but this is not true for simulants that move into college GQ during the simulation". Also, removing the emphasis from "*college*" in the previous clause.

Couldn't directly suggest these changes on GitHub because those lines weren't edited.